closure/compiler/test/BUILD

```diff
@@ -33,7 +33,7 @@ closure_js_binary(
 
 file_test(
     name = "minification",
-    content = "console.log(\"hello world\");\n",
+    content = "(function(){console.log(\"hello world\");}).call(this);\n",
```

Comment:
Are you looking to disable the output_wrapper for this test? If so, do you prefer `output_wrapper = None`, `output_wrapper = ""` or `output_wrapper = "%output%"`?